cmd/nerdctl/volume_rm.go

```diff
@@ -45,21 +43,12 @@ func newVolumeRmCommand() *cobra.Command {
 }
 
 func volumeRmAction(cmd *cobra.Command, args []string) error {
-	client, ctx, cancel, err := newClient(cmd)
-	if err != nil {
-		return err
-	}
-	defer cancel()
-	containers, err := client.Containers(ctx)
-	if err != nil {
-		return err
-	}
```

Comment:
Why this refacto ? We may use these variables `client, ctx, cancel, err` in a futur improvement. It is better to keep them at this place

Reply:
I remove it first, off topic